stylesheets/_structure.forms.scss

```diff
@@ -8,9 +8,7 @@
 
 // each 'block' of a form, usually consisting of a label and one or more inputs
 .form__group {
-    display: table;
     margin-bottom: .5em;
-    //table-layout: fixed; // prevent resizable textareas from causing the column to grow in width
```

Comment:
As discussed, with `table-layout: fixed;` already been commented out I'll create another ticket to explore/fix.